dogs.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt A Dog</title>
+  </head>
+  <body>
+    <header>
+      <figure class="company-logo">
+        <img src="images/adoptadog-logo.svg" alt="Company Logo">
+      </figure>
+
+      <figure class="cart-icon">
+        <img src="images/cart-icon.svg" alt="Cart Icon">
+      </figure>
+
+      <nav>
+        <ul>
+          <li class="navlist"><a href="index.html">Home</a></li>
+          <li class="navlist"><a href="dogs.html">Dogs</a></li>
+          <li class="navlist"><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <main>
+
+        <h1>Here are all of our dogs!</h1>
+
+      <section class="dog-photos">
```

Comment:
You're on the right path for these tiles. You will want to have some sort of wrapping element around all of the dog tiles to lay them out.